RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,12 +8,165 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State var isOnToggle = false
+    @State var isAddingBalanceAlertShown = false
+    @State var balance = 0
+    @State var addedMainBalanceValue = ""
+    @State var addedAdditionalBalanceValue = ""
+    @State var isVisibleAccount = false
+    @State var isTransferAlertShown = false
+    
+    @State var newAccount = AccountManagement()
+    
     var body: some View {
         VStack {
-            Text("")
+            ZStack {
+                HStack {
+                    VStack(spacing: 15) {
+                        
+                        Text("Main Account\nBalance")
+                            .font(.title2)
+                            .multilineTextAlignment(.center)
+
+                        Text("\(balance)")
+                            .foregroundColor(.red)
+                        
+                        Button {
+                            isAddingBalanceAlertShown = true
+                        } label: {
+                            Text("Increase balance")
+                        }
+                        .buttonStyle(.bordered)
+                        .alert(Text("Add to Main balance"),
+                               isPresented: $isAddingBalanceAlertShown) {
+                            TextField("Value", text: $addedMainBalanceValue)
+                                .keyboardType(.numberPad)
+                            
+                            Button {
+                                balance += Int(addedMainBalanceValue) ?? 0
+                                addedMainBalanceValue = ""
+                            } label: {
+                                Text("OK")
```

Comment:
```suggestion
                                Text("OK")
```
В общем, везде где по тексту ниже есть Text с стрингой внутри вынеси в константы плз

Reply:
в константы